src/main/java/com/solodive/championselect/service/dto/riotapi/RiotExtendedSummonerDTO.java

```diff
@@ -0,0 +1,44 @@
+package com.solodive.championselect.service.dto.riotapi;
+
+import java.util.Arrays;
+
+public class RiotExtendedSummonerDTO {
+
+    private RiotBasicSummonerDTO riotBasicSummonerDTO;
+    private RiotSummonerRankDTO[] riotSummonerRankDTO;
```

Comment:
riotSummonerRankDTO - zrezygnowaliśmy z tablic na rzecz list (wyjątek - jak faktycznie inaczej się nie da, bo tak wynika z mapowania)?

Reply:
No chyba tak wynika z mapowania, ale skoro to jest problem, to zawsze mogę w serwisie zwracać Arrays.asList :rocket: